tests/src/index.js

```diff
@@ -9,6 +9,7 @@ import { StacksMainnet } from '@stacks/network'
 nock.disableNetConnect()
 
 bskConfig.network = new StacksMainnet()
+bskConfig.network.blockstackAPIUrl = bskConfig.network.coreApiUrl
```

Comment:
wouldn't this be easy to change  the env var to `STACKS_API_URL` etc, vs continuing to use the `BLOCKSTACK_API_URL`?

Reply:
Maybe, but it's not a high enough priority for us to invest more resources in at the moment.